public/index.html

```diff
@@ -6,8 +6,169 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <title>Untitled</title>
         <link rel="author" href="humans.txt">
+        <link> <link rel="stylesheet" type="text/css" href="facesmashmain.css">
+    
+    <title>"Facesmash"</title>          
+    
     </head>
+    
     <body>
-        <h1>Audacious Aretes</h1>
+        <ul class="fixedheader clearfix">
+            <li><h1><a href = "/">Facesmash</a></h1></li>
+                <li>
+                <ul>
+                    <li><a href ="/">About Me</a></li>
+                    <li><a href ="/">Sign Up</a></li>
+                    <li><a href ="/">Login</a></li>
+                </ul>
+            </li>
+        </ul>
```

Comment:
Just going to comment that there is something wrong here...I'll let you decide what it is.